cerberus-tables/src/ast/mod.rs

```diff
@@ -135,7 +135,7 @@ impl<'md> Ident<'md> {
         self.0
     }
 
-    /// Returns the textual content of the wrapped Span.
+    /// Returns the textual content of the wrapped [`Span'].
```

Comment:
Should be `` [`Span`] ``; note both are backticks. Rustdoc parses these automatically.